src/tests/unit/services/UserInfoRequestProcessor.test.ts

```diff
@@ -0,0 +1,319 @@
+import { UserInfoRequestProcessor } from "../../../services/UserInfoRequestProcessor";
+import { Metrics } from "@aws-lambda-powertools/metrics";
+import { mock } from "jest-mock-extended";
+import { Logger } from "@aws-lambda-powertools/logger";
+import { MISSING_AUTH_HEADER_USERINFO, VALID_USERINFO } from "../data/userInfo-events";
+import { BavService } from "../../../services/BavService";
+import { Response } from "../../../utils/Response";
+import { HttpCodesEnum } from "../../../models/enums/HttpCodesEnum";
+import { ISessionItem } from "../../../models/ISessionItem";
+import { PersonIdentityItem } from "../../../models/PersonIdentityItem";
+import { absoluteTimeNow } from "../../../utils/DateTimeUtils";
+import { MockFailingKmsSigningJwtAdapter, MockKmsJwtAdapter } from "../utils/MockJwtVerifierSigner";
+import * as Validations from "../../../utils/Validations";
+
+/* eslint @typescript-eslint/unbound-method: 0 */
+/* eslint jest/unbound-method: error */
+
+let userInforequestProcessorTest: UserInfoRequestProcessor;
+const mockBavService = mock<BavService>();
+let mockSession: ISessionItem;
+let mockPerson: PersonIdentityItem;
+const passingKmsJwtAdapterFactory = (_signingKeys: string) => new MockKmsJwtAdapter(true);
+const failingKmsJwtAdapterFactory = (_signingKeys: string) => new MockKmsJwtAdapter(false);
+const failingKmsJwtSigningAdapterFactory = (_signingKeys: string) => new MockFailingKmsSigningJwtAdapter();
```

Comment:
This doesn't look like it's being used

Reply:
Yea copied all 3 across, removed failingKmsJwtSigningAdapterFactory